Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN sudo apt-get update \
 COPY requirements.txt .
 RUN sudo pip install --no-cache-dir -r requirements.txt
 
+COPY requirements_new.txt .
```

Comment:
I recommend you don't use two different requirements files. merge the two together and run install on just the one